fastlane/Fastfile

```diff
@@ -101,13 +106,16 @@ lane :release do |options|
   Dir.chdir(get_root_folder) do
     sh(args)
   end
+  Dir.chdir(File.expand_path('react-native-purchases-ui', get_root_folder)) do
```

Comment:
feel free to ignore, but we could update the description of this lane to mention that it publishes both packages